images/kamu-base-git/Dockerfile

```diff
@@ -0,0 +1,86 @@
+FROM ubuntu:focal AS builder
+
+ARG KAMU_BRANCH=master
+
+
+WORKDIR /app/
+
+ENV PATH=/root/.cargo/bin:$PATH
+
+RUN apt update  \
+    && apt install -y  \
+      git  \
+      curl \
+      gcc \
+    && git clone https://github.com/kamu-data/kamu-cli  \
+      --branch $KAMU_BRANCH  \
+      --depth 1 \
+      /app/ \
+    && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs  \
+      | sh -s -- --default-toolchain nightly -y \
+    && cargo build -p kamu-cli
+
+################################################################################
+
+FROM ubuntu:focal
+
+# Podman
+# Source: https://github.com/containers/podman/blob/056f492f59c333d521ebbbe186abde0278e815db/contrib/podmanimage/stable/Dockerfile
+RUN apt update && \
+    apt -y install curl wget gnupg unzip jq && \
+    . /etc/os-release && \
+    echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /" \
+    | tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list && \
+    curl -L "https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/Release.key" \
+    | apt-key add - && \
+    apt update && \
+    apt install ca-certificates && \
+    apt -y install podman fuse-overlayfs && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
90% of build time for `kamu-base` image is installing podman, aws cli, etc.

I wonder if for these dev builds we could just use `FROM kamu-data/kamu-base:latest` that comes with all this stuff in it, and we only replace the `kamu` binary.

This would significantly speed up the builds and save space too.

Reply:
Totally makes sense

Reply:
Done